### PR TITLE
Google Stadia Support

### PR DESCRIPTION
**Pull request by tomspilman**
These are the initial bits needed in the public repo for Google Stadia support.

### CODE REVIEW

File: MonoGame.Framework.Content.Pipeline/MonoGame.Framework.Content.Pipeline.csproj [hunk 1]
```diff
@@ -262,4 +262,6 @@ This package provides you with the content pipeline for Windows, Mac and Linux w
     </Compile>
   </ItemGroup>
 
+  <Import Project="..\Stadia\Pipeline.targets" Condition="exists('..\Stadia\Pipeline.targets')"/>
```
Comment (tomspilman) [inline MonoGame.Framework.Content.Pipeline/MonoGame.Framework.Content.Pipeline.csproj]: This seems like the right way to go for all future console support.  Lets us modify the project without it being visible to the public.

File: MonoGame.Framework/Graphics/Shader/Shader.cs [hunk 2]
```diff
@@ -113,7 +113,7 @@ internal Shader(GraphicsDevice device, BinaryReader reader)
                 Attributes[a].location = reader.ReadInt16();
             }
 
-            PlatformConstruct(isVertexShader, shaderBytecode);
+            PlatformConstruct(Stage, shaderBytecode);
```
Comment (tomspilman) [inline MonoGame.Framework/Graphics/Shader/Shader.cs]: This was a convenient change which seemed good for future support of other shader stages.

File: MonoGame.Framework/Graphics/SurfaceFormat.cs [hunk 3]
```diff
@@ -36,7 +36,7 @@ public enum SurfaceFormat
         /// <summary>
         /// DXT5. Texture format with compression. Surface dimensions must be a multiple 4.
         /// </summary>
-        Dxt5 = 6,
+        Dxt5,
```
Comment (tomspilman) [inline MonoGame.Framework/Graphics/SurfaceFormat.cs]: No functional or numeric change to the enum values...   this was just a mistake that wasn't necessary.  Makes interop with native code more clear.

File: MonoGame.Framework/Utilities/PlatformInfo.cs [hunk 4]
```diff
@@ -38,6 +38,8 @@ public static MonoGamePlatform MonoGamePlatform
                 return MonoGamePlatform.PlayStation4;
 #elif PSVITA
                 return MonoGamePlatform.PSVita;
+#elif STADIA
+                return MonoGamePlatform.Stadia;
```
Comment (tomspilman) [inline MonoGame.Framework/Utilities/PlatformInfo.cs]: @Jjagg @harry-cpp - May be good to clean up these #ifs and replace it with partial class stuff?

Comment (harry-cpp) [reply MonoGame.Framework/Utilities/PlatformInfo.cs]: This is one of those rare moments where I think #if is what we actually want.

**Comment by tomspilman** [comment]
@Jjagg @harry-cpp - Please review and merge this as soon as you can.

File: MonoGame.Framework/Utilities/PlatformInfo.cs (hunk 4 above)
Comment (Jjagg) [reply MonoGame.Framework/Utilities/PlatformInfo.cs]: Yeah, we chose to do it like this. Code is fairly recent.

**Comment by Jjagg** [comment]
Looks good, merging!

**Pull request merged**